src/Posts/Post.jsx

```diff
@@ -248,15 +286,44 @@ return (
     )}
     {!state.hasBeenFlagged && (
       <Post>
+        <>
+          {repostData ? (
+            <Widget
+              src="${REPL_ACCOUNT}/widget/AccountProfileOverlay"
+              props={{
+                accountId,
+                profile,
+                children: (
+                  <div className="row">
+                    <div className="col-auto ">
+                      <i className="bi bi-repeat" /> reposted by{" "}
+                      <TextLink href={repostedByProfileUrl} ellipsis>
+                        @{repostData.reposted_by}
+                      </TextLink>
+                      {tags.length > 0 && (
+                        <TagsWrapper>
+                          <Widget src="${REPL_ACCOUNT}/widget/Tags" props={{ tags, scroll: true }} />
+                        </TagsWrapper>
+                      )}
+                    </div>
+                  </div>
+                ),
+                placement: props.overlayPlacement,
+                verifications,
+                showFlagAccountFeature,
+              }}
+            />
+          ) : null}
+        </>
         <Header>
           <div className="row">
             <div className="col-auto">
               <Widget
                 src="${REPL_ACCOUNT}/widget/AccountProfile"
                 props={{
-                  profile,
+                  profile: repostData ? null : profile,
```

Comment:
I think we can also add a relationship with `accounts` table to pass a profile here as well. Any objections?

Such a relationship has been added to `notifications` and feeds table (need to check for the correct spelling of that table)

Reply:
@gabehamilton or @charleslavon any thoughts on this? Maybe something for us to look into for another task?

Reply:
That's a good callout @shelegdmitriy however with the reduced focus on social features and upcoming gateway change, we should not spend that additional effort for now.

Reply:
Sounds reasonable